### PR TITLE
release: v0.6.0

### PR DESCRIPTION
**Pull request by tiborsimko**
Signed-off-by: Tibor Šimko <tibor.simko@cern.ch>

### CODE REVIEW

File: CHANGES.rst [hunk 1]
```diff
@@ -1,39 +1,61 @@
...
-
-  - Allows to explore workflow results by running interactive Jupyter notebook
-    sessions on the workspace files.
-  - Allows to declare computing resources needed for workflow runs, such as
-    access to CVMFS repositories.
-  - Improves ``reana-client`` command-line client with new options to stop
-    workflows, diff workflows, move and remove files.
-  - Upgrades CWL engine to 1.0.20181118133959.
-  - See additional changes in `reana-client release notes <https://reana-client.readthedocs.io/en/latest/changes.html#version-0-5-0-2019-04-24>`_.
+    - Adds support for HTCondor compute backends for all workflow engines.
```
Comment (diegodelemos) [inline CHANGES.rst]: Maybe `workflow engine` is a bit too much our jargon? What do you think about something more neutral like this or similar:

"Adds [HTCondor](https://research.cs.wisc.edu/htcondor/) support for all workflow languages."

File: CHANGES.rst [hunk 2]
```diff
@@ -1,39 +1,61 @@
...
+    - Adds support for Slurm compute backends for all workflow engines.
+    - Allows to run hybrid analysis pipelines when various parts of the workflow run on various HTCondor, Kubernetes and Slurm compute backends.
```
Comment (diegodelemos) [inline CHANGES.rst]: What do you think about this or similar? (just removing `various` dup, adding more punctuation and links):
"Allows running hybrid analysis pipelines, where different parts of the workflow can run on various compute backends: [HTCondor](https://research.cs.wisc.edu/htcondor/), [Kubernetes](https://kubernetes.io/) and [Slurm](https://slurm.schedmd.com/documentation.html)."

File: CHANGES.rst (hunk 1 above)
Comment (tiborsimko) [reply CHANGES.rst]: Reformulated adding the workflow names at the end.

File: CHANGES.rst (hunk 2 above)
Comment (tiborsimko) [reply CHANGES.rst]: Yes, it reads better, reformulated -- but kept two "different" occurrences for wanted stylistic effect

**Pull request merged**